src/main/resources/jupyter/init-actions.sh

```diff
@@ -145,6 +147,11 @@ if [[ "${ROLE}" == 'Master' ]]; then
     mkdir /certs
     chmod a+wx /work
 
+    # shared jupyter/welder dir
```

Comment:
Q: create a new dir like I'm doing here, or reuse an existing Jupyter dir?

Reply:
Per previous discussions with @Qi77Qi , the plan-of-record is to mount the shared volume directly onto the Jupyter working directory. I believe this is currently `/home/user/work` in `jupyter-docker-compose.yaml`, which appears to currently occupied by `/work`.

Perhaps just continue using the existing /work directory, but start managing it with Welder?

Reply:
That makes sense to me.  I'll update it to do that.